jenkins/opensearch-dashboards/Jenkinsfile

```diff
@@ -62,7 +62,12 @@ pipeline {
                                 architecture: "x64"
                             )
                             assembleManifest()
-                            uploadArtifacts(upload: true)
+                            uploadArtifacts(
+                                upload: true, 
+                                dirs: ['builds', 'dist'], 
+                                paths: ['builds/manifest.yml', 'dist/manifest.yml'],
```

Comment:
Derive paths for the notification from `dirs`, just pass the name of the index file, e.g. `index_file: manifest.yml` or `index_file: index.html`.

Reply:
For build we have the path to the manifest file but for tests we have path till `test-results` directory.

Reply:
I think the two arrays is throwing me off. Then maybe something like this?

```
uploadArtifacts(
   upload: true
   artifacts: [
        {
           dir: 'builds',
           path: 'builds/manifest.yml',
        },
        {
           dir: 'builds',
           path: 'dist/manifest.yml',
        },
   ]
```